### PR TITLE
hotfix/v0.17.4

### LINKED ISSUE #161: Hotfix: PR template
**Summary**
Much like ISSUE_TEMPLATE, we should have PR templates to include common steps, e.g., version bump, Linked issues/PRs.

**Objectives**
1. Have one for features
2. Have one for hotfixes
3. Have one for releases

**Proposal**

Copy the PR templates over from ReQUIAM:
https://github.com/ualibraries/ReQUIAM/issues/120

*Testing notes*

*Additional notes*

NA

**Implemented in**: hotfix/161_pr_templates

### PR DESCRIPTION
**Pull request by damian-romero**
Fixes #161

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,7 +240,8 @@ Currently, there are two GitHub Action workflows:
 A list of released features and their issue number(s).
 List is sorted from moderate to minor revisions for reach release.
 
-v0.17.0 - v0.17.3:
+v0.17.0 - v0.17.4:
+ * New pull request templates #161
  * Include Travis CI configuration (disabled see #136) #129
  * Include GitHub Actions for Python CI build and testing #136
  * Add script for curation folder rename #120
```
Comment (astrochun) [inline README.md:247]:
```suggestion
 * Include Travis CI configuration (disabled see #136) #129
 * Include GitHub Actions for Python CI build and testing #136
 * Add script for curation folder rename #120
 * Minor: New pull request templates #161
```

**Review by astrochun — APPROVED** [review]
Looks good. Note that changelog is structured to indicate big changes over small changes in order.

File: README.md (hunk 1 above)
Comment (damian-romero) [reply README.md]: Perfect :) 👍

**Review by astrochun — APPROVED** [review]
Changes look good. Will merge.

**Pull request merged**